From 110ef8e0304dd253072bf4000173fafaf36b4416 Mon Sep 17 00:00:00 2001
From: Sen Jiang <senj@google.com>
Date: Wed, 18 Jul 2018 17:27:24 -0700
Subject: [PATCH 2/8] Fix loading ueventd.${ro.hardware}.rc.

Regression introduced in aosp/717324.

Bug: 111543389
Test: device boots further
Change-Id: I4cf57381104aa1a801cf82a42b1c5ae1a2273e89
Merged-In: I4cf57381104aa1a801cf82a42b1c5ae1a2273e89
(cherry picked from commit d76f174a785d2f1c17999a2d23b1fea2a33e4b1e)

Former-commit-id: 94d81965da56566f900ccffc12e7a45c9e0ea0ab
Former-commit-id: 624610ea654815d6fbfc5f1e40067a165d5055c1
Former-commit-id: fc2e262a2c11eef90be55e54648d026c0f10d707
---
 init/ueventd.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/init/ueventd.cpp b/init/ueventd.cpp
index 264e136af4..a5f9e2b25d 100644
--- a/init/ueventd.cpp
+++ b/init/ueventd.cpp
@@ -248,7 +248,8 @@ int ueventd_main(int argc, char** argv) {
         auto hardware = android::base::GetProperty("ro.hardware", "");
 
         auto ueventd_configuration =
-                ParseConfig({"/ueventd.rc", "/vendor/ueventd.rc", "/odm/ueventd.rc", hardware});
+                ParseConfig({"/ueventd.rc", "/vendor/ueventd.rc", "/odm/ueventd.rc",
+                             "/ueventd." + hardware + ".rc"});
 
         device_handler = DeviceHandler{std::move(ueventd_configuration.dev_permissions),
                                        std::move(ueventd_configuration.sysfs_permissions),
-- 
2.17.1

